cardano-api/src/Cardano/Api.hs

```diff
@@ -565,6 +565,8 @@ module Cardano.Api (
     -- ** Conversions
     --TODO: arrange not to export these
     toNetworkMagic,
+    fromAlonzoRdmrPtr,
+    renderScriptWitnessIndex,
```

Comment:
Put this next to the export of `ScriptWitnessIndex` itself. It is not a conversion.